src/todos.js

```diff
@@ -0,0 +1,48 @@
+export default class Todos {
```

Comment:
[OPTIONAL] Since the previous reviewer didn't notice it I will keep this optional. But for this project, you need to create an array and render it. CRUD functions shouldn't have been implemented yet.
<img width="1056" alt="Screenshot 2023-04-07 at 11 52 58 PM" src="https://user-images.githubusercontent.com/25887253/230677770-f3f636a1-7358-48ed-ba35-49847ec9281b.png">